mlflow/tracking/client.py

```diff
@@ -452,15 +453,15 @@ def delete_traces(
             request_ids=request_ids,
         )
 
-    def get_trace_info(self, request_id: str) -> TraceInfo:
+    def get_trace(self, request_id: str) -> Trace:
         """
-        Get the trace matching the `request_id`.
+        Get the trace matching the specified ``request_id``.
 
         Args:
-            request_id: String id of the trace to fetch.
+            request_id: String ID of the trace to fetch.
 
         Returns:
-            The fetched Trace object, of type ``mlflow.entities.TraceInfo``.
+            The retrieved :py:class:`Trace`.
```

Comment:
```suggestion
            The retrieved :py:class:`Trace <mlflow.entities.Trace>`.
```
I think this is where Sphinx complains

Reply:
Thanks! :D